VTEX - Catalog API.json

```diff
@@ -12859,7 +12927,7 @@
                     "Sales channel"
                 ],
                 "summary": "Get sales channel by ID",
-                "description": "Retrieves a specific sales channel by its ID. \r\n\r\n## Response body example\r\n\r\n```json\r\n{\r\n    \"Id\": 1,\r\n    \"Name\": \"Loja Principal\",\r\n    \"IsActive\": true,\r\n    \"ProductClusterId\": null,\r\n    \"CountryCode\": \"BRA\",\r\n    \"CultureInfo\": \"pt-BR\",\r\n    \"TimeZone\": \"E. South America Standard Time\",\r\n    \"CurrencyCode\": \"BRL\",\r\n    \"CurrencySymbol\": \"R$\",\r\n    \"CurrencyLocale\": 1046,\r\n    \"CurrencyFormatInfo\": {\r\n        \"CurrencyDecimalDigits\": 1,\r\n        \"CurrencyDecimalSeparator\": \",\",\r\n        \"CurrencyGroupSeparator\": \".\",\r\n        \"CurrencyGroupSize\": 3,\r\n        \"StartsWithCurrencySymbol\": true\r\n    },\r\n    \"Origin\": null,\r\n    \"Position\": 2,\r\n    \"ConditionRule\": null,\r\n    \"CurrencyDecimalDigits\": 1\r\n}\r\n```",
+                "description": "Retrieves a specific sales channel by its ID.\r\n\r\n## Permissions\r\n\r\nAny user or [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys) must have at least one of the appropriate [License Manager resources](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3) to be able to successfully run this request. Otherwise they will receive a status code `403` error. These are the applicable resources for this endpoint:\r\n\r\n| **Product** | **Category** | **Resource** |\r\n| --------------- | ----------------- | ----------------- |\r\n| Catalog API | General | View Product |\r\n\r\nThere are no applicable [predefined roles](https://help.vtex.com/en/tutorial/predefined-roles--jGDurZKJHvHJS13LnO7Dy) for this resource list. You must [create a custom role](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#creating-a-role) and add at least one of the resources above in order to use this endpoint. To learn more about machine authentication at VTEX, see [Authentication overview](https://developers.vtex.com/docs/guides/authentication).\r\n\r\n>\u2757 To prevent integrations from having excessive permissions, consider the [best practices for managing app keys](https://help.vtex.com/en/tutorial/best-practices-application-keys--7b6nD1VMHa49aI5brlOvJm) when assigning License Manager roles to integrations.",
```

Comment:
Is this permission correct? I think there are more specific permissions related to trade policies (looking at [the resources doc](https://help.vtex.com/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3)), maybe it's worth checking with the team

Reply:
It is correct, I have checked with the Catalog team spreadsheet with all the resources documented: https://docs.google.com/spreadsheets/d/1M7CdgmpGxB6bWJIS0MIF4pnxPJWRe2cPu7eu-4cGJK0/edit?gid=1263733726#gid=1263733726